### PR TITLE
PR for #288

### PR DESCRIPTION
**Pull request by hirako2000**
Fixes Text  and adds a basic example for it

### CODE REVIEW

File: src/components/meshes/Text.js [hunk 1]
```diff
@@ -117,8 +119,6 @@ class Text extends MeshComponent {
       });
     });
 
-    super.wait(promise);
```
Comment (hirako2000) [inline src/components/meshes/Text.js]: me no understand this, why wait for promise here?

Comment (sasha240100) [inline src/components/meshes/Text.js]: @hirako2000 This was used. Now if build returns a `Promise` - [mesh will automatically wait for it](https://github.com/WhitestormJS/whs.js/blob/dev/src/core/MeshComponent.js#L110).

File: examples/basic/text/script.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  }),
+
+  controlsModule
+]);
+
+const text = new WHS.Text({
+  text: 'Gentilis Bold Text',
+
+  parameters: {
+      font: `${process.assetsPath}/fonts/gentilis_bold.typeface.json`,
```
Comment (sasha240100) [inline examples/basic/text/script.js]: Can we move it to:

```js
const text = new WHS.Text({
  text: 'Gentilis Bold Text',
  url: '/path/to/font.typeface.json'
});
```

And if dev wants to input a ready font:

```js
const text = new WHS.Text({
  text: 'Gentilis Bold Text',
  font: myFont
});
```

So the algorythm will use `url` if it is provided, if not - then check for `font`.

Comment (hirako2000) [inline examples/basic/text/script.js]: then it would not return a promise, would it?

Comment (sasha240100) [inline examples/basic/text/script.js]: @hirako2000 if `font` - then returns a `Mesh`, if `url` - then a `Promise` that resolves with a Mesh

Comment (hirako2000) [inline examples/basic/text/script.js]: Improvement will be another issue. I've put back super.wait()

**Pull request merged**